.github/workflows/prbuild.yaml

```diff
@@ -19,11 +19,11 @@ jobs:
         # image to use) for each kubernetes_version value.
         include:
           - kubernetes_version: "kubernetes:latest"
-            node_image: "docker.io/kindest/node:v1.20.2"
+            node_image: "stevesloka/kind-node:v1.21.0"
```

Comment:
nit: Add TODO comment for reverting this

Reply:
👍  I opened #3610 as well.